input/otlp/logs.go

```diff
@@ -106,18 +106,22 @@ func (c *Consumer) convertInstrumentationLibraryLogs(
 ) {
 	otelLogs := in.LogRecords()
 	for i := 0; i < otelLogs.Len(); i++ {
-		event := c.convertLogRecord(otelLogs.At(i), baseEvent, timeDelta)
+		event := c.convertLogRecordWithScope(otelLogs.At(i), in.Scope(), baseEvent, timeDelta)
```

Comment:
Instead of passing the scope into this method, could we call `translateScopeMetadata(in.Scope(), baseEvent)` here. That should also generalise well to traces and metrics, when we're ready to make that change.

Reply:
I have thought about it and chose this instead of doing another `baseEvent.CloneVT`. Otherwise, this `translateScopeMetadata` will overwrite fields in the shared baseEvent at resource level.

Reply:
OK, I think that's fair. If the cost of processing instrumentation scope ever outweighs the clone, then we could revisit it.